tracee-ebpf/main.go

```diff
@@ -1,9 +1,12 @@
+// +build go1.16
```

Comment:
I suppose it's not a huge deal but are we ok with requiring go 1.16? There's probably a way to use embed only if go1.16 but not otherwise, like having the embed directive in a separate file

Reply:
Yeah I guess it's safe to leave it out. This is what happens without it (on go 1.15 or earlier):

```
GOOS=linux GOARCH=amd64 CC=clang CGO_CFLAGS="-I /tracee/tracee-ebpf/dist/libbpf/usr/include" CGO_LDFLAGS="/tracee/tracee-ebpf/dist/libbpf/libbpf.a" go build -v -o dist/tracee-ebpf \
-ldflags "-X main.version=v0.5.0-20-gd58ae46"
go: downloading github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
go: downloading github.com/urfave/cli/v2 v2.1.1
go: downloading github.com/aquasecurity/tracee/libbpfgo v0.0.0-20210115081842-487d1e44fcda
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d
go: downloading github.com/russross/blackfriday/v2 v2.0.1
go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
main.go:4:2: package embed is not in GOROOT (/usr/local/go/src/embed)
```

Which is fine, it's still a compile time error at the end of the day.